src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IEventReferenceOperation.cs

```diff
@@ -201,5 +201,153 @@ public static void M()
             VerifyOperationTreeAndDiagnosticsForTest<MemberAccessExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
 
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
```

Comment:
What about adding or removing delegate from events? Or is that going to be a separate PR?

Reply:
Yes, I plan to handle `IEventAssignmentOperation` next.